SpatialGDK/Documentation/content/get-started/build-unreal-fork.md

```diff
@@ -4,7 +4,7 @@ As the GDK extends Unreal Engine's networking capabilities at its core, to use i
 
 ### Unreal Engine EULA
 
-In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, and agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github).
+In order to get access to our fork, you need to link your GitHub account to a verified Epic Games account, agree to the Unreal Engine End User License Agreement ([EULA](https://www.unrealengine.com/en-US/eula)) and accept the invite to join the [EpicGames organisation on Github](https://github.com/EpicGames). You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github). When you are done you will receive an email invite to join the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork.
 
```

Comment:
Addition to "accept the invite" super useful.

These two final sentences:
"When you are done you will receive an email invite to join the EpicGames organisation on Github. You need to be a member in order to access the Unreal Engine fork."
Makes it seem like there are extra steps on top of those already listed. I had to think about whether there were extra steps. I think we can lose these last two sentences?

Reply:
Aah, I see, good call. It's a potential stumbling block. Let's take your idea and be bossier...

"You will not be able to use the GDK without doing this first. To do this, see the [Unreal documentation](https://www.unrealengine.com/en-US/ue4-on-github) and make sure you accept the email invitation from EpicGames."